backends/graphics/opengl/opengl-sys.h

```diff
@@ -60,22 +57,16 @@
 
 #ifdef __ANDROID__
 	#include <GLES/gl.h>
```

Comment:
why keep Android out of GLAD?

Reply:
The Android port doesn't (yet) support dynamically loading OpenGL functions. It would be nice to support this at some point to allow switching between GLESv1 and GLESv2 at runtime, but for now I'd prefer to keep this PR simple.

Reply:
we are going using only GLESv2 for Android, see 3D backend PR. Android NDK require devices with GLESv2 support.